packages/grid/x-data-grid/src/hooks/features/columnHeaders/useGridColumnHeaders.tsx

```diff
@@ -268,20 +300,215 @@ export const useGridColumnHeaders = (props: UseGridColumnHeadersProps) => {
       );
     }
 
+    return (
+      <GridColumnHeaderRow role="row" aria-rowindex={headerGroupingMaxDepth + 1}>
+        {columns}
+      </GridColumnHeaderRow>
+    );
+  };
+
+  const getColumnsHeaderGroups = (params?: {
```

Comment:
What do you think?

```suggestion
  const getColumnGroupHeaders = (params?: {
```

Reply:
Yes, sounds more consistent with other namings